### ISSUE TITLE
Configurable Navbar location

### ISSUE THREAD

**Issue by brianwthomas** [bootstrapstudio/bootstrap-better-nav#2]
It would be really nice to have a option to configure the navbar pop-out from the left or right side of the screen. Currently, when I use this, it only defaults to scrolling out from right to left, and I'd like to see it pop-out from the left side towards the right.

**Comment by Areahints** [comment]
Hello I need this functionality too @brianwthomas do you have a workaround for now?

**Comment by danny-mark** [comment]
Hi guys, thank you for the comments! Sorry it took me so long to respond.

I just added the functionality to change the placement of the menu. It's super easy to do, just add the `better-bootstrap-nav-left` class to the Bootstrap navbar:

```html
<nav class="navbar navbar-expand-md better-bootstrap-nav-left">
```
Commit 61c69fc closes #2

**Comment by Areahints** [comment]
Better late than never! Thanks
